config/registry_image_pruner/image_pruner/test_prune_images.py

```diff
@@ -364,7 +394,34 @@ def test_remove_tags_nothing_to_remove(self, delete_image_tag):
         delete_image_tag.assert_not_called()
 
     @patch("prune_images.delete_image_tag")
-    def test_remove_tags_multiple_tags(self, delete_image_tag):
+    @patch("prune_images.manifest_exists")
+    def test_remove_tags_nothing_to_remove_digest_exists(self, manifest_exists, delete_image_tag):
+        tags = [
+            {
+                "name": "sha256-502c8c35e31459e8774f88e115d50d2ad33ba0e9dfd80429bc70ed4c1fd9e0cd.att",
+                "manifest_digest": "sha256:125c1d18ee1c3b9bde0c7810fcb0d4ffbc67e9b0c5b88bb8df9ca039bc1c9457",
+            },
+            {
+                "name": "sha256-502c8c35e31459e8774f88e115d50d2ad33ba0e9dfd80429bc70ed4c1fd9e0cd.sbom",
+                "manifest_digest": "sha256:351326f899759a9a7ae3ca3c1cbdadcc8012f43231c145534820a68bdf36d55b",
+            },
+        ]
+
+        manifest_exists.side_effect = [
+            True,
+            True,
+        ]
+
+        with self.assertRaisesRegex(AssertionError, expected_regex="no logs of level INFO"):
+            with self.assertLogs(LOGGER) as logs:
+                remove_tags(tags, QUAY_TOKEN, "some", "repository", dry_run=True)
```

Comment:
It does not make sense to set `dry_run=True` here. When it is True, `delete_image_tag` will be never called under whatever conditions.

Reply:
well it was in that other test above :)